Add documentation on subtitles

Arnei opened the pull request:
*This PR aims at contributing to the implementation of #4407, but touches on just one of the aspects to realize #4407's vision. It should therefore likely not be merged alone, but alongside other PRs.*

Adds a new documentation file about subtitles.

Arnei committed:
Add documentation on subtitles

Adds a new documentation file about subtitles.

KatrinIhler commented on docs/guides/admin/docs/modules/subtitles.md:
```diff
@@ -0,0 +1,79 @@
+
+The following is a list of tags that Opencast components may recognize:
+- `lang:<language>`
+	- `<language>` represents a language code of the form TODO: WHICH LANGUAGE CODE?
```
I assume these Todos should be done before we merge this?

Arnei replied:
Thanks for the reminder.

Regarding language codes: I don't have a solid opinion on which to use. Do you? I should probably bring this is up in the Technical Meeting.

Regarding no major language: Might just have to ignore that for now and cross that bridge when we come to it?

KatrinIhler replied:
> Regarding language codes: I don't have a solid opinion on which to use. Do you?

I think I prefer including the region in the language code, so `en-GB` instead of `en`. Most people commenting on the issue also seemed to think so.

> I should probably bring this is up in the Technical Meeting.

Good idea.

> Regarding no major language: Might just have to ignore that for now and cross that bridge when we come to it?

Agreed.

Arnei committed:
Add language code spec to docs

Removes Todos from the subtitles docs, and adds the IETF
BCP 47 language tag spec as the default we want to support.
Most technical meeting folks were in favor of it, and
my research indicates that this indeed seems to be best spec
for our use case.

mliradelc commented on docs/guides/admin/docs/modules/subtitles.md:
```diff
@@ -0,0 +1,81 @@
+This document records guidelines and conventions on how subtitles should be handled in Opencast.
+
+<div class=warn>
+These guidelines are new to Opencast 14. Not all parts of Opencast may yet adhere to them! 
```
```suggestion
These guidelines are new to Opencast 15. Not all parts of Opencast may yet adhere to them! 
```

mliradelc replied:
Or maybe rewording to avoid update every time OC upgrades.

Arnei replied:
Thanks for catching that again :see_no_evil:  
Maybe it's actually good having to look at this every major OC upgrade, so we're forced the re-evaluate if this warning is still appropriate?

mliradelc replied:
I don't know, It's straightforward easy to forget to update the tiny details in the docs. I just found in the dev docs some repos to Bitbucket that are from almost 5 years deprecated.

Arnei replied:
Well, in that case the version number at least serves as an indicator that this part of the docs is likely outdated, and as a user you should proceed with care :D

mliradelc replied:
Fair Enough :)

mliradelc reviewed (approved):
Docs are ok and very easy to undestand. Only a little update regarding opencast version

Arnei committed:
Fix OC version number in subtitles docs

Since we missed OC14, it's now OC15.
